.github/workflows/build.yml

```diff
@@ -59,6 +59,11 @@ jobs:
     timeout-minutes: 10
     runs-on: ubuntu-18.04
     steps:
+      - name: Setup Go 1.13
```

Comment:
Looks like this commit contains same code as in PR - https://github.com/hyperledger/aries-framework-go/pull/658/files.

Reply:
I mentioned in the description `Depends on PR #658`